zephyr/module.yml

```diff
@@ -13,10 +13,10 @@ blobs:
     doc-url: https://github.com/telink-semi/zephyr_hal_telink_b91_ble_lib/blob/9a45494713524fe8ad3b5f98b7606e39c8d82517/README.md
 
   - path: liblt_9528_zephyr.a
-    sha256: 0b30dc56fbdada609d2ad8481563c41da3857190582b6d65f6310053110fec02
+    sha256: aa4e101f38662dfcffb2f7821a6a8f89bbfc7245af416597a7792ee2aa183eaf
     type: lib
     version: 'V4.0.1.0_Patch_0004'
```

Comment:
Please increase the version. Otherwise it's difficult fo figure out that we have changes.